.prettierrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  bracketSpacing: false,
+  jsxBracketSameLine: true,
+  singleQuote: true,
+  trailingComma: 'all',
+  semi: false,
+  arrowParens: 'avoid',
+};
```

Comment:
Why using a .js file? Isn't a .json simpler/more appropriate?

Reply:
both options are completely valid. I used a js because it was like that in the migration tool. I don't think it is a big difference

Reply:
@gndelia my preference is to use a `.json` file when the we just need to export a static object. Seem simpler to me.